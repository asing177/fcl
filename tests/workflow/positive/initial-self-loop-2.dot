digraph workflow {
edge [color="#000000"]
graph [bgcolor=transparent]


initial [shape=point; width=0.3; style=filled; color="#0e64ce"; label=""]
terminal [shape=point; width=0.3; peripheries=2; style=filled; color="#d11010"; label=""]

1[label=<<FONT POINT-SIZE="16">go1</FONT><FONT POINT-SIZE="10" COLOR="blue"> </FONT>>shape=box; fontname="Arial"; style=filled; color=black; fillcolor=gray75;]
2[label=<<FONT POINT-SIZE="16">go2</FONT><FONT POINT-SIZE="10" COLOR="blue"> </FONT>>shape=box; fontname="Arial"; style=filled; color=black; fillcolor=gray75;]

initial -> 1 ; 1 -> initial
initial -> 2 ; 2 -> terminal



}
