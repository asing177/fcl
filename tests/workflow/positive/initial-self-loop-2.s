@initial
go1() { stay() }

@initial
go2() { terminate() }
