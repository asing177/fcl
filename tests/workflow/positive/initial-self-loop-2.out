 
Method arguments and effects:
go1: () -> {write via stay}
go2: () -> {write via terminate}